apps/condo/domains/user/components/containers/AuthLayoutContext.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { createContext, useCallback } from 'react'
+import { useMutation } from '@core/next/apollo'
+import { useAuth } from '@core/next/auth'
+import { useIntl } from '@core/next/intl'
+import { useAntdMediaQuery } from '@condo/domains/common/utils/mediaQuery.utils'
+import { runMutation } from '@condo/domains/common/utils/mutations.utils'
+import { SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION, SIGNIN_MUTATION } from '@condo/domains/user/gql'
+
+interface IAuthLayoutContext {
+    isMobile: boolean
+    signInByEmail: ({ email, password }) => Promise<unknown>,
+    signInByPhone: ({ phone, password }) => Promise<unknown>,
+}
+
+export const AuthLayoutContext = createContext<IAuthLayoutContext>({
+    isMobile: false,
+    signInByEmail: ({ email, password }) => null,
+    signInByPhone: ({ phone, password }) => null,
+})
+
+export const AuthLayoutContextProvider: React.FC = (props) => {
+    const intl = useIntl()
+    const { refetch } = useAuth()
+
+    const [signinByPhoneMutation] = useMutation(SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION)
+    const [signinByEmailMutation] = useMutation(SIGNIN_MUTATION)
+
+    const colSize = useAntdMediaQuery()
+
+    const signInByPhone = useCallback(({ phone, password }) => {
+        // @ts-ignore TODO(Dimitreee): remove after runMutation typo
+        return runMutation({
+            mutation: signinByPhoneMutation,
+            variables: { phone, password },
+            onCompleted: () => {
+                refetch()
+            },
+            intl,
+        }).catch(error => {
+            console.error(error)
```

Comment:
User notification ?

Reply:
@pahaz i'm not sure i knew how exactly and what about notify user, guess @dkoviazin can help's here.